GitForWindowsHelper/slash-commands.js

```diff
@@ -174,7 +198,10 @@ module.exports = async (context, req) => {
                 )
 
                 const answer = await triggerBuild()
-                const answer2 = await appendToComment(`The workflow run [was started](${answer.html_url})`)
+                const answer2 = await appendToComment(aarch64Answer
+                    ? `The [i686/x86_64](${answer.html_url}) and the [amd64](${aarch64Answer.html_url}) workflow runs were started.`
```

Comment:
Did you mean `arm64` instead of `amd64`? 😊

Reply:
🤦 yes, I did ;-)